rbi/yast/wfm.rbi

```diff
@@ -0,0 +1,24 @@
+# typed: strong
+module Yast::WFM
+  class NoParameter; end
+  YCPAny = T.type_alias do
+    T.any(
+      NilClass,
+      T::Boolean,
+      Integer,
+      Float,
+      Symbol,
+      String,
+      T::Array[T.untyped],
```

Comment:
cannot we have recursion here like `T::Array[YCPAny]`

Reply:
Let's try... nope.